src/main/java/seedu/address/logic/parser/ArchiveCommandParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ArchiveCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+import java.util.List;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
```

Comment:
This import statement should be right after the package statement to follow the code standard.